SERVER/test/MODELS/create.js

```diff
@@ -0,0 +1,76 @@
+const {Pool}= require("pg");
+const dotenv= require("dotenv");
+
+dotenv.config();
+
+const pool= new Pool({
+    connectionString: process.env.DATABASE_URL
+});
+
+pool.on('connect', () => {
+    //console.log('connected to the db');
+});
+
+//Create all tables
```

Comment:
No need to use unnecessary comments. Well named variables are a better way of documenting your code than comments

Use comments for confusing code that needs explaining, or code which when altered will cause some side effect.

Reply:
I have removed unnecessary comments, renamed variable to be better recognized and used ES6 import and export instead of require